.github/workflows/Test.yml

```diff
@@ -0,0 +1,34 @@
+name: Tests
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: [1.5]
+        os: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v1.0.0
+      - uses: azure/login@v1.1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+      - name: Run Az CLI script
+        run: |
+          az group create -l southcentralus -n test_azstorage
+          az storage account create -n testazstoragewxyz -g test_azstorage -l southcentralus
```

Comment:
We should prob variablize some of the things we talked about like the rsg name and the SA name. I have a simple version working here for reference: https://github.com/ChevronETC/AzManagers.jl/blob/github-cicd/.github/workflows/Tests.yml

(See the az group create/delete commands)